tests/unit/lastuser_core/test_model_client_AuthCode.py

```diff
@@ -0,0 +1,38 @@
+from lastuserapp import db
+import lastuser_core.models as models
+from .test_db import TestDatabaseFixture
+from datetime import datetime, timedelta
+
+class TestAuthCode(TestDatabaseFixture):
+
+    def test_AuthCode_init(self):
+        """Test to verify creation of AuthCode instance"""
+        crusoe = self.fixtures.crusoe
+        client = self.fixtures.client
+        auth_code = models.AuthCode(user=crusoe, client=client, redirect_uri=u'http://batdogadventures.com/fun', scope=u'id')
```

Comment:
Atleast in the init test, test all the values set. Including `redirect_uri` and `scope`.